plugins/importer/src/main/java/com/managemc/importer/data/cache/EssentialsPunishmentsCache.java

```diff
@@ -0,0 +1,11 @@
+package com.managemc.importer.data.cache;
+
+import com.managemc.importer.reader.model.EssentialsMute;
+import com.managemc.importer.translation.EssentialsTypeTranslator;
+
+public class EssentialsPunishmentsCache extends VanillaPunishmentsCache {
```

Comment:
It was silly to call this a cache, because it's definitely not a cache. Oh well. This class temporarily stores punishments formatted the way ManageMC expects before they are imported.